api/v1beta1/ssp_webhook.go

```diff
@@ -159,3 +167,16 @@ func validateOperandPlacement(namespace string, placement *api.NodePlacement) er
 
 	return clt.Create(context.TODO(), deployment, &client.CreateOptions{DryRun: []string{metav1.DryRunAll}})
 }
+
+// TODO: also validate DataImportCronTemplates in general once CDI exposes its own validation
```

Comment:
Maybe the validation could do a dry-run create API call, to test if `DataImportCron` is valid. But maybe it would time out if there are a lot of `DataImportCronTemplates` specified.

Reply:
CDI plans on exporting the actual validation code and as we already consume their API we can just use that once they do.